src/receivers/ExpressReceiver.ts

```diff
@@ -500,7 +500,7 @@ export function verifySignatureAndParseBody(
   return parseRequestBody(body, contentType);
 }
 
-function buildBodyParserMiddleware(logger: Logger): RequestHandler {
+export function buildBodyParserMiddleware(logger: Logger): RequestHandler {
```

Comment:
Exported this once more for ease of testing. Another option is to move this method to a separate module in its own file. Let me know if anyone has a preference.

Reply:
Just exporting it looks fine!